### ISSUE TITLE
firedoc install will not install npm dependency for theme

### ISSUE THREAD

**Issue by nantas** [fireball-x/firedoc#35]
``` bash
Cloning into 'firedoc-theme-notab'...
remote: Counting objects: 360, done.
remote: Compressing objects: 100% (66/66), done.
remote: Total 360 (delta 33), reused 0 (delta 0), pack-reused 291
Receiving objects: 100% (360/360), 174.24 KiB | 94.00 KiB/s, done.
Resolving deltas: 100% (211/211), done.
Checking connectivity... done.
installing the npm dependencies for firedoc-theme-notab
env: node: No such file or directory
firedoc-theme-notab installed at /Users/nantas/.nvm/versions/node/v0.12.0/lib/node_modules/firedoc/themes/firedoc-theme-notab
```

Everytime I run `firedoc install firedoc-theme-notab`

**Comment by yorkie** [comment]
Here I used the following command:

```
$ firedoc install https://github.com/fireball-x/firedoc-theme-notab
```

and got

```
Cloning into 'firedoc-theme-notab'...
remote: Counting objects: 362, done.        
remote: Compressing objects: 100% (68/68), done.        
remote: Total 362 (delta 34), reused 0 (delta 0), pack-reused 291        
Receiving objects: 100% (362/362), 174.42 KiB | 7.00 KiB/s, done.
Resolving deltas: 100% (212/212), done.
Checking connectivity... done.
installing the npm dependencies for firedoc-theme-notab
underscore@1.8.3 node_modules/underscore
firedoc-theme-notab installed at /Users/yorkie/workspace/firebox/firedoc/themes/firedoc-theme-notab
```

because I have not made a url rule for converting `firedoc-theme-notab` to its github link, @nantas could you make a double check in exact url way? Thanks

**Comment by yorkie** [comment]
Will introduce programatic git and npm package instead of bash shell to address the issue, doing...

**Comment by yorkie** [comment]
@nantas Sorry for the late fix, but we have this fix now, feel free to try with `firedoc@1.9.0`, but there might be some notable changes on new version:
1. because we haven't used the bash script to clone theme project and install dependencies, so I think this bug should not be reproduced, if still exists, tell me please.
2. because I introduced `nodegit` and `npm` into the dependencies of FireDoc, and even the `nodegit` requires a C++ Addon, it bindings the libgit C library and download the **binary**, so I'm quite sure `v1.9.0` would take more to install.
3. (**improvement**) supported to install by name just like:

``` sh
$ firedoc install notab
$ firedoc install firedoc-theme-notab
$ firedoc install https://github.com/fireball-x/firedoc-theme-notab
```

Both are working correctly :)
